common/source/docs/common-nooploop.rst

```diff
@@ -0,0 +1,87 @@
+.. _common-nooploop:
```

Comment:
should not have common in name...its not a common doc....abd file should be renamed also....suggest using only  nooploop as name

Reply:
Hi @Hwurzburg Thank you. I see every file inside common/source/docs start with `common`. If this file does not start with `common`, it may looks different from others?

Reply:
Sorry, I thought it was in copter source too early in the morning no coffee just remove it from the landing page and then add it to the table of contents on the master's feature page